counterpartylib/lib/api.py

```diff
@@ -694,6 +746,68 @@ def handle_post():
             _set_cors_headers(response)
             return response
 
+        ######################
+        # HTTP REST API
+        ######################
+        # Handle blocks route.
+        @app.route('/rest/block/<block_hash>.<format>', methods=["GET",])
+        def handle_get_block(block_hash, format):
+            # Check for tx_hash validity.
```

Comment:
Yes, but the db contains no private information (I guess it could leak the db file path, or something?). :) With v9.49.4, the server doesn't even know about wallets/privkeys. Of course, it's a good idea in any case.